sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1220,11 +1288,14 @@ def process_bundle(self, inputs, expected_outputs):
     # Actually start the bundle.
     if registration_future and registration_future.get().error:
       raise RuntimeError(registration_future.get().error)
-    process_bundle = beam_fn_api_pb2.InstructionRequest(
+
+    # Runner requests the SDK to fulfill the following instruction:
```

Comment:
I'm not sure whether I understand this comment correctly, but from what I know so far, the runner sends ProcessBundleRequest to sdk and waiting for the response.

Reply:
```suggestion
```